Task_TheFlexGame/index.css

```diff
@@ -1 +1,60 @@
-/* Start coding below for FLEX GAME TASK*/
+/* Start coding below for FLEX GAME TASK*/

```

Comment:
The asterisk (*) in line 2 should have a forward slash (/) before it to properly comment out the line.